src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -23,16 +35,53 @@ public HttpClientHandlerTest(ITestOutputHelper output)
         }
         
         [Fact]
-        public void SendAsync_SimpleGet_Success()
+        public async void SendAsync_SimpleGet_Success()
+        {
+            using (var handler = new HttpClientHandler())
+            {
+                using (var client = new HttpClient(handler))
+                {
+                    // TODO: This is a placeholder until GitHub Issue #2383 gets resolved.
+                    HttpResponseMessage response = await client.GetAsync(HttpTestServers.RemoteGetServer);
+                    Assert.Equal(HttpStatusCode.OK, response.StatusCode);
+                    string responseContent = await response.Content.ReadAsStringAsync();
+                    _output.WriteLine(responseContent);
```

Comment:
There's nothing about the data in responseContent we want to be verifying / asserting?